### DISCUSSION TITLE
`go install` doesn't pull down all files

### DISCUSSION THREAD

**Discussion by glennzw** [community/community#34352] [Programming Help]
Hi all,

When running the following command:

`go install github.com/gophish/gophish@latest`

This [folder](https://github.com/gophish/gophish/tree/master/static/js/src/vendor) is not pulled down:

`static/js/src/vendor` 

After digging around it seems to be that vendor folders are reserved:

https://github.com/golang/go/issues/48332

Previously with this project we had used `go get github.com/gophish/gophish`, but as that has been deprecated I updated our documentation to use `go install`, but now realise it fails with the above vendor folder issue.

The only solution I can think of is to use: `git clone https://github.com/gophish/gophish.git`

Any advice?

G

**Comment by samus-aran** [comment]
Welcome to the GitHub Community, @glennzw ! :wave: We're happy you're here. You are more likely to get a useful response if you are posting your question(s) in the applicable category and someone might give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!

**Comment by kecci** [comment]
Hi @glennzw 

I think your best way to get your vendor is using `go mod vendor`

To generate the vendor, command on terminal:
```sh
go mod init
go mod vendor
```

Hope this help your question
